console/console-init/ui/src/graphql-module/queries/iot_project.ts

```diff
@@ -78,16 +78,10 @@ export const CREATE_IOT_PROJECT = gql`
   }
 `;
 
-const ENABLE_IOT_PROJECTS = gql(
-  `mutation enable_iotProjects($a:[ObjectMeta_v1_Input!]!) {
-    enableIotProjects(input:$a)
+const TOGGLE_IOT_PROJECTS_STATUS = gql(
+  `mutation projectsstatustoggle($a: [ObjectMeta_v1_Input!]!, $status: Boolean!){
```

Comment:
mutation query name is not readable as projectsstatustoggle. can you try this toggle_iot_project_status.

Reply:
done